Student/Student/Submissions/TextSubmission/TextSubmissionPresenter.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright (C) 2019-present Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import Core
+
+protocol TextSubmissionViewProtocol: class {
+    func showError(_ error: Error)
+    func dismiss(animated flag: Bool, completion: (() -> Void)?)
```

Comment:
We have an `EmptyHandler` typealias if you prefer for `() -> Void`. (knit pick) (i'm not sure where EmptyHandler lives is good, just saying, i probably need to move it somewhere logical)

Reply:
Does it matter that I'm making it match the one from UIViewController? Would it still be appropriate to use our alias?